src/main/java/org/jabref/cli/Launcher.java

```diff
@@ -42,11 +42,11 @@
  * - Start the JavaFX application (if not in cli mode)
  */
 public class Launcher {
-    private static Logger LOGGER;
-    private static String[] ARGUMENTS;
+    private static Logger logger;
```

Comment:
This looks wrong, static vars should be always upper case

Reply:
Launcher.java is the only exception where we need to initialize the logger with some magic. See https://github.com/koppor/jabref/blob/106dba1cd37ccd6c66ae22e209649490a7d17b67/src/main/java/org/jabref/cli/Launcher.java#L92. Back then, I tried multiple other ways to initialize the logger, but this one looked best to me!

When deciding to be either consistent with variable casing everywhere or have an exception here (to prevent WTF during short code-browsing), I decided for the former.

Reply:
No, I mean this in general, it seemed to have an impact on all other classes

Reply:
I checked all final, non-static variables at [`106dba1` (#641)](https://github.com/koppor/jabref/pull/641/commits/106dba1cd37ccd6c66ae22e209649490a7d17b67). All non-static variables should be camel-cased - even though they are final?

Reply:
Recipe "documentation": https://docs.openrewrite.org/recipes/java/cleanup/renameprivatefieldstocamelcase